resources/views/template/navbar.blade.php

```diff
@@ -180,48 +185,55 @@
                                 data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                 <i class="fas fa-bell fa-fw"></i>
                                 <!-- Counter - Alerts -->
-                                <span class="badge badge-danger badge-counter">3+</span>
+                                <span class="badge badge-danger badge-counter">{{$unreadNotifications}}</span>
```

Comment:
Can you hide this indicator when there are no unread notifications?
![image](https://user-images.githubusercontent.com/32955000/171083487-8b2f9d89-5278-4af1-a1f3-5c8061d268a8.png)